### PR TITLE
chore: Changelog for Kong Gateway 3.4.0.0

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Changelog for the 3.4.0.0 release, based on the OSS and EE changelogs.

Also adding changelogs for any new plugin features that weren't yet documented.

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by lena-larionova** [comment]
Link failures are all github 429s.

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -5,6 +5,332 @@ no_version: true
...
+Starting with Kong Gateway 3.4.0.0, Kong is not building new Ubuntu 18.04
+images or packages, and Kong will not test package installation on Ubuntu 168.04.
+
+    If you need to install Kong Gateway on Ubuntu 18.04, see the documentation for
+    [previous versions](/gateway/3.3.x/install/linux/ubuntu/).
+* Vitals is now off by default. To enable it, set `vitals=on` in Kong configuration.
+* Amazon Linux 2022 artifacts are renamed to Amazon Linux 2023, based on AWS's own renaming.
+* LMDB encryption has been disabled. The option `declarative_config_encryption_mode` has been removed from `kong.conf`.
+* The `/consumer_groups/:id/overrides` endpoint is deprecated in favor of a more generic plugin scoping mechanism. 
+See the new [consumer groups]() entity.
```
Comment (Guaris) [inline app/gateway/changelog.md:30]:
```suggestion
See the new [consumer groups](https://developer.konghq.com/spec/937dcdd7-4485-47dc-af5f-b805d562552f/25d728a0-cfe3-4cf4-8e90-93a5bb15cfd9#/consumer_groups/get-consumer_groups) entity.
```

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -5,6 +5,332 @@ no_version: true
 
 <!-- vale off -->
 
+## 3.4.0.0
+**Release Date** TBA
+
+### Breaking changes and deprecations
+
+* **Cassandra DB support removed:** Cassandra DB support has been removed. 
+It is no longer supported as a datastore for Kong Gateway.
```
Comment (Guaris) [inline app/gateway/changelog.md:14]:
```suggestion
It is no longer supported as a data store for Kong Gateway. 
```

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/gateway/changelog.md [hunk 3]
```diff
@@ -5,6 +5,332 @@ no_version: true
...
+  [#11090](https://github.com/Kong/kong/pull/11090)
+* Removed Kong branding from the kong HTML error template.
+  [#11150](https://github.com/Kong/kong/pull/11150)
+
+#### Plugins
+
+* Validation for plugin queue related parameters has been
+  improved. [#10840](https://github.com/Kong/kong/pull/10840)
+  * `max_batch_size`, `max_entries`, and `max_bytes` are now
+  integers instead of numbers.
```
Comment (Guaris) [inline app/gateway/changelog.md:115]:
```suggestion
  * `max_batch_size`, `max_entries`, and `max_bytes` are now declared
  as `integer` not `number`. 
```

File: app/gateway/changelog.md [hunk 4]
```diff
@@ -5,6 +5,332 @@ no_version: true
...
+
+* [**Kafka Log**](/hub/kong-inc/kafka-log/) (`kafka-log`)
+  * The Kafka Log plugin now supports the `custom_fields_by_lua` configuration for 
+  dynamic modification of log fields using Lua code.
+
+* [**GraphQL Rate Limiting Advanced**](/hub/kong-inc/graphql-rate-limiting-advanced/) (`graphql-rate-limiting-advanced`)
+  * The `host` field of this plugin now accepts Kong upstream targets.
+
+* [**OpenTelemetry**](/hub/kong-inc/opentelemetry/) (`opentelemetry`)
+  * Introduced support for the AWS X-Ray propagation header
```
Comment (Guaris) [inline app/gateway/changelog.md:136]:
```suggestion
  * Introduced support for the AWS X-Ray propagation header.
```

File: app/gateway/changelog.md [hunk 5]
```diff
@@ -5,6 +5,332 @@ no_version: true
...
+  * Fixed an issue where the plugin always validated the request body even 
+  if the method spec had no `requestBody` defined.
+  * Fixed an issue where the comparison between large absolute value numbers could be incorrect 
+  due to the number being converted to exponential notation.
+
+* [**Request Validator**](/hub/kong-inc/request-validator/) (`request-validator`)
+  * Optimized the response message for invalid requests.
+
+* [**ACME**](/hub/kong-inc/acme/) (`acme`)
+ * Fixed an issue where the sanity test didn't work with `kong` storage in hybrid mode.
```
Comment (Guaris) [inline app/gateway/changelog.md:286]:
```suggestion
    * Fixed an issue where the sanity test didn't work with `kong` storage in hybrid mode.
```

**Review by Guaris — CHANGES_REQUESTED** [review]

**Review by Guaris — APPROVED** [review]

File: app/gateway/changelog.md [hunk 6]
```diff
@@ -5,6 +5,332 @@ no_version: true
...
+
+* [**OAuth 2.0 Introspection**](/hub/kong-inc/oauth2-introspection/) (`oauth2-introspection`)
+  * Fixed an issue where the plugin failed when processing a request with JSON that is not a table.
+
+* [**gRPC Gateway**](/hub/kong-inc/grpc-gateway/) (`grpc-gateway`)
+  * Fixed an issue where an array with one element would fail to be encoded.
+  * Fixed an issue where empty (all default value) messages couldn't be unframed correctly.
+  [#10836](https://github.com/Kong/kong/pull/10836)
+
+* * [**Response Transformer**](/hub/kong-inc/response-transformer/) (`response-transformer`) and
```
Comment (lena-larionova) [inline app/gateway/changelog.md:248]:
```suggestion
* [**Response Transformer**](/hub/kong-inc/response-transformer/) (`response-transformer`) and 
```

**Pull request merged**